controllers/provisioners/ekscloudformation/ekscloudformation.go

```diff
@@ -54,6 +54,10 @@ const (
 	UnrecoverableDeleteErrorString = "UnrecoverableDeleteError"
 )
 
+const (
+	reqPolicies = "AmazonEKSWorkerNodePolicy,AmazonEKS_CNI_Policy,AmazonEC2ContainerRegistryReadOnly"
```

Comment:
I think it may be best for this to be in the scope of `GetManagedPolicyARNs` instead of top-level for this package, as it's the only function that will need to access it.
Also, consider changing to `requiredPolicies` for readability.

Regarding the type being a comma separated string, doesn't it make more sense as a slice? e.g.
```
requiredPolicies = []string{"AmazonEKSWorkerNodePolicy", "AmazonEKS_CNI_Policy" ,"AmazonEC2ContainerRegistryReadOnly"}
```
That way you don't have to `names := strings.Split(reqPolicies, ",")` below since it will be already iterable.

Reply:
Agreed! Let me change that